aiotfm/player.py

```diff
@@ -230,7 +230,7 @@ def __init__(self, packet: Packet):
 		for _ in range(packet.read8()):
 			self.orbs.add(packet.read8())
 
-		packet.readBool()
+		self.isonline = packet.readBool()

```

Comment:
would probably name it is_online instead just for consistency

Reply:
Oh my bad. Well akschually the rest of the naming follows camel case so isOnline?